dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/model/ProviderMethodModel.java

```diff
@@ -44,8 +45,8 @@ public String getMethodName() {
         return methodArgTypes;
     }
 
-    public String getServiceName() {
-        return serviceName;
+    public ConcurrentMap<String, Object> getAttributeMap() {
+        return attributeMap;
```

Comment:
return the mutable member variable ** attributeMap** may be harmful as any caller can have the reference and modify it. may be we can provide put and get method for attributeMap in this class it self.